resources/ui/adw/prefs.ui

```diff
@@ -214,39 +214,132 @@ SPDX-License-Identifier: GPL-3.0-or-later
     </child>
   </object>
 
+
   <object class="AdwPreferencesPage" id="general-prefs">
+    <property name="margin-top">20</property>
+    <property name="margin-bottom">20</property>
+    <property name="margin-start">20</property>
+    <property name="margin-end">20</property>
```

Comment:
I think these are not required.

```suggestion
```

Reply:
ummm i think these were in the main code, so i wasn't exactly sure if it was needed.
👍